doc/Architecture/arc-tec_x-road_technologies.md

```diff
@@ -79,7 +83,7 @@ Table 1. Technology matrix of the X-Road
  JRuby 9                            |                     | X                  |                         |
  Ubuntu 18.04                       | X                   | X                  | X                       | X
  Red Hat Enterprise Linux 7 (RHEL7) | X                   |                    |                         | X
- PostgreSQL 9.4                     |                     | X\[[1](#Ref_1)\]   |                         |
+ Red Hat Enterprise Linux 8 (RHEL8) | X                   |                    |                         | X
```

Comment:
Updated so that Java 8 is still mentioned for the central server, but elsewhere Java 11 is used. See https://github.com/nordic-institute/X-Road/pull/1038/commits/e34158a1b2da2c62f1cca1202883ae0637acb02c